src/main/resources/com/google/api/codegen/java/http_unit_test.snip

```diff
@@ -0,0 +1,139 @@
+@extends "java/common.snip"
+@extends "java/initcode.snip"
+
+@snippet generate(xapiTest)
+  {@renderFileHeader(xapiTest.fileHeader)}
+
+  @@javax.annotation.Generated("by GAPIC")
+  public class {@xapiTest.testClass.name} {
+    private static final MockHttpService MOCK_SERVICE = new MockHttpService();
```

Comment:
Where is this defined? (it's not checked into GAX yet)

Reply:
Oh yes, just opened the gax-java PR at https://github.com/googleapis/gax-java/pull/492.